app/controllers/orders_controller.rb

```diff
@@ -21,4 +31,10 @@ def find_order
       flash[:notice] = "Please login appropriately to view that order."
       redirect_to login_path
   end
+
+  def create_item_orders
+    @cart.contents.each do |item, qty|
```

Comment:
Sounds good. I'm not sure which model this should live in as this is being called on the Order model, but we're creating a new ItemOrder. Maybe we can ask our project sponsor during our 7/31 meeting.